manageiq-automation_engine.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,lib}/**/*", "LICENSE.txt", "Rakefile", "README.md"]
 
-  s.add_dependency "rubyzip", "~>1.2.2"
+  s.add_dependency("rubyzip", "~>1.2.2")
 
-  s.add_development_dependency "codeclimate-test-reporter", "~> 1.0.0"
-  s.add_development_dependency "simplecov"
+  s.add_development_dependency("codeclimate-test-reporter", "~> 1.0.0")
+  s.add_development_dependency("simplecov")
```

Comment:
@astrozzc This is usually treated as a DSL so we can leave it unchanged.

Would be good to look at adding `add_dependency`, `add_development_dependency` and even `add_runtime_dependency` to the robocop `IgnoredMethods` list here: https://github.com/ManageIQ/guides/blob/master/.rubocop_base.yml#L94